bin/travis-build.bash

```diff
@@ -5,6 +5,7 @@ echo "This is travis-build.bash..."
 
 echo "Installing the packages that CKAN requires..."
 sudo apt-get update -qq
+sudo apt-get install xmlsec1
```

Comment:
@avdata99 you need to also install `libxmlsec1-dev`, so that pip can build xmlsec.